src/components/ChartJsWrapper/index.jsx

```diff
@@ -73,6 +73,10 @@ const ChartJsWrapper = ({
         textAlign: 'center',
         width: spinnerSize,
       }}>
+      <ErrorPanel
+        className={classes.errorPanel}
+        error="Something went wrong, please try again later."
```

Comment:
You can store this message in a variable such as `const defaultErrorMessage` and put it into a constants or helpers file since you are using it more than once.